Mathlib/CategoryTheory/Yoneda.lean

```diff
@@ -788,23 +828,6 @@ theorem coyonedaPairing_map (P Q : C × (C ⥤ Type v₁)) (α : P ⟶ Q) (β :
     (coyonedaPairing C).map α β = coyoneda.map α.1.op ≫ β ≫ α.2 :=
   rfl
 
-variable {C} in
-/-- A bijection `(coyoneda.obj X ⋙ uliftFunctor ⟶ F) ≃ F.obj (unop X)` which is a variant
-of `coyonedaEquiv` with heterogeneous universes. -/
-def coyonedaCompUliftFunctorEquiv (F : C ⥤ Type max v₁ w) (X : Cᵒᵖ) :
```

Comment:
Could you add a `deprecated alias` for this definition, towards `uliftCoyonedaEquiv`?